x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/service_list/index.tsx

```diff
@@ -153,9 +154,27 @@ export function getServiceColumns({
       ? [
           {
             field: ServiceInventoryFieldName.HealthStatus,
-            name: i18n.translate('xpack.apm.servicesTable.healthColumnLabel', {
-              defaultMessage: 'Health',
-            }),
+            name: (
+              <>
+                {i18n.translate('xpack.apm.servicesTable.healthColumnLabel', {
+                  defaultMessage: 'Health',
+                })}{' '}
```

Comment:
Do we need this `{' '}`?

Reply:
Yes, otherwise there won't be any space between the title and the icon.